### DISCUSSION TITLE
Web3Wallet documentation

### DISCUSSION THREAD

**Discussion by phipsae** [WalletConnect/walletconnect-monorepo#4076] [Developer Support]
Dear Web3Wallet team,

I hope this message finds you well. 
I am currently in the process of integrating the web3wallet into my custom-built wallet, aiming to enhance its connectivity with various dapps.
However, I've encountered a bit of a challenge due to the complexities in the implementation phase, as I find the existing documentation not entirely clear for my needs.
Could you kindly assist by pointing me towards a more detailed tutorial or example that outlines the necessary steps for this integration? A step-by-step guide or a hands-on example would be extremely helpful.

Best regards,
Philip

**Comment by glitch-txs** [comment]
Hi, which framework are you using? we have some examples in the resources section https://docs.walletconnect.com/web3wallet/resources

**Comment by phipsae** [comment]
Thanks for reaching out!
I'm using next.js. Although I've checked out the available examples, I would find it much more helpful to have some guidance that includes detailed instructions and explanations.
